packages/ui/src/components/primitives/message/Embed.tsx

```diff
@@ -56,8 +56,17 @@ export const Embed = (props: EmbedProps) => {
 			) : (
 				<EmbedText text={embed.author?.name} />
 			)}
-
-			<EmbedText className="text-xl font-bold" text={embed.title} />
```

Comment:
this is a bug, an embed could have a title or a provider name

Reply:
The provider is like what the site is that is giving the data for the link embed 👍